doc/templates.rst

```diff
@@ -227,7 +232,7 @@ the values you pass as arguments:
 
     {# versus #}
 
-    {{ data|convert_encoding(from='iso-2022-jp', to='UTF-8') }}
+    {{ data|convert_encoding(from: 'iso-2022-jp', to: 'UTF-8') }}
```

Comment:
As you replaced the `=` with `:` in this doc, does that mean using `:` will (sooner or later) be the "recommanded" syntax ?  

(something we wondered with @VincentLanglet to chose a potential "default choice" in Twig-CS-Fixer rule)

Reply:
I think we will support both for the foreseeable future. I would recommend using `:` for a CS fixer though.